### PR TITLE
Tests: make hardcoded domain configurable and use RFC-compliant @example.com

### PR DESCRIPTION
**Pull request by MatthiasWinkelmann**
Hello,

I felt that I had probably send you enough spam and made the domain in integration tests configurable. 

I also changed the e-mail addresses in tests from, i. e. tony@stark.com to tony.stark@example.com. I felt uncomfortable with the possibility of them leaking into production code and possibly interfering with real people.

I hope these changes are useful. Thx!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ after_script:
   - MIX_ENV=docs mix inch.report
 notifications:
   recipients:
-    - steve@stevedomin.com
+    - steve@stevedomin.com
```
Comment (princemaple) [inline .travis.yml]: that space is probably a mistake?

Comment (MatthiasWinkelmann) [inline .travis.yml]: Yes, sorry.

File: test/integration/adapters/postmark_test.exs [hunk 2]
```diff
@@ -6,18 +6,18 @@ defmodule Swoosh.Integration.Adapters.PostmarkTest do
   @moduletag integration: true
 
   setup_all do
-    config = [api_key: System.get_env("POSTMARK_API_KEY")]
+    config = [api_key: System.get_env("POSTMARK_API_KEY"), config: System.get_env("POSTMARK_DOMAIN")]
```
Comment (stevedomin) [inline test/integration/adapters/postmark_test.exs]: s/config/domain

**Comment by stevedomin** [comment]
Hi @MatthiasWinkelmann, thanks for the PR! (And sorry for the late review, I'm on holidays :))

One minor comment and this is good to go!

**Comment by stevedomin** [comment]
Thank you so much @MatthiasWinkelmann and sorry for the wait!

**Pull request merged**